ts/packages/node/package.json

```diff
@@ -52,6 +53,7 @@
     "typeorm": "^0.3.16"
   },
   "devDependencies": {
+    "@types/ini": "^1.3.31",
     "@types/lodash": "^4.14.195"
   }
-}
+}
```

Comment:
Please keep the whitespace here, this difference isn't necessary to your contribution 🙏